src/Postmark/PostmarkClient.php

```diff
@@ -106,18 +106,25 @@ function sendEmailWithTemplate($from, $to, $templateId, $templateModel, $inlineC
 		$body['TrackOpens'] = $trackOpens;
 		$body['Attachments'] = $attachments;
 		$body['TemplateModel'] = $templateModel;
-		$body['TemplateId'] = $templateId;
 		$body['InlineCss'] = $inlineCss;
 		$body['Metadata'] = $metadata;
-		
-		
+
+
 		// Since this parameter can override a per-server setting
 		// we have to check whether it was actually set.
 		// And only include it in the API call if that is the case.
 		if ($trackLinks !== NULL) {
 			$body['TrackLinks'] = $trackLinks;
 		}
 
+		if ( is_int( $templateIdOrAlias ) ) {
```

Comment:
Should this be `is_numeric` instead of `is_int`? My code happened to store template integers inside strings, and it took me an hour just now to figure out what was going on. This code was putting the template ID into TemplateAlias, and those templates didn't have aliases.